source/_integrations/voip.markdown

```diff
@@ -0,0 +1,114 @@
+---
+
+The VoIP integration enables users to talk to [Assist](/docs/assist) using an analog phone and a VoIP adapter such as the [Grandstream HT801](https://www.grandstream.com/products/gateways-and-atas/analog-telephone-adaptors/product/ht801).
+
+<p class='img'>
+  <img src="/images/integrations/voip/voip_adapter.png" />
+  Connecting a phone to Home Assistant requires an adapter.
+</p>
+
+## Getting Started
+
+Start by setting up the VoIP integration. In Home Assistant, go to *Settings*, *Devices & Services*, clicking the *Add Integration* button, and entering "voip":
+
+<p class='img'>
+  <img src="/images/integrations/voip/voip_install.jpg" />
+  The VoIP integration must be installed before you can call Home Assistant.
+</p>
+
+Select "Voice over IP" and confirm the installation. Next, configure your Grandstream VoIP adapter to call Home Assistant.
+
+## Grandstream Configuration
+
+The Grandstream HT801 and HT802 have been tested with the VoIP integration. To connect your phone to Home Assistant, start by logging into your Grandstream device's web interface (default password: "admin"):
+
+<p class='img'>
+  <img src="/images/integrations/voip/grandstream_login.jpg" />
+  Login page of the Grandstream HT801/HT802 VoIP adapter.
+</p>
+
+Navigate to the port configuration page. On the HT801 there will only be a single port to configure, while the HT802 has two ports:
+
+<p class='img'>
+  <img src="/images/integrations/voip/grandstream_port_config.jpg" />
+  Port configuration page for the HT802. The HT801 only has a single port.
+</p>
+
+Locate the "Offhook Auto-Dial" setting and enter `*47<IP>*5060` where `<IP>` is the IP address of your Home Assistant server using `*` instead of `.` between octets. For example, the IP `192.168.1.100` would be entered as `*47192*168*1*100*5060`:
+
+<p class='img'>
+  <img src="/images/integrations/voip/grandstream_autodial.jpg" />
+  Enable offhook auto-dial, so your phone automatically calls Home Assistant when you pick it up.
+</p>
+
+Save your settings at the bottom of the page by clicking the "Apply" button:
+
+<p class='img'>
+  <img src="/images/integrations/voip/grandstream_apply.jpg" />
+  You must apply the settings before they will take effect.
+</p>
+
+## Test Call
+
+If everything is configured correctly, pick up the phone and you should hear Home Assistant speak the following message:
+
+> This is your smart home speaking. Your phone is connected, but you must configure it within Home Assistant.
+
+If you don't hear the message, ensure that your Grandstream is connected to the network and that your phone is connected to the Grandstream. See the [troubleshooting](#troubleshooting) section for other suggestions.
+
+## Allowing Calls
+
+Calls from new devices are blocked by default since voice commands could be used to control sensitive devices, such as locks and garage doors. Ensure that you've configured [Assist](/docs/assist/) to only expose the entities you want to be controllable via voice.
```

Comment:
```suggestion
Calls from new devices are blocked by default since voice commands could be used to control sensitive devices, such as locks and garage doors. Ensure you've configured [Assist](/docs/assist/) only to expose the entities you want to be controllable via voice.
```